WebDriverAgentLib/Utilities/FBXCodeCompatibility.m

```diff
@@ -23,15 +23,6 @@
 
 @implementation XCUIApplication (FBCompatibility)
 
-+ (instancetype)fb_applicationWithPID:(pid_t)processID
-{
-  if (0 == processID) {
```

Comment:
Are you sure this protection is not needed? What happens if the requested pid is 0?